Sources/SwiftLexicalLookup/LookupName.swift

```diff
@@ -225,4 +266,27 @@ import SwiftSyntax
   ) -> [LookupName] {
     [.declaration(namedDecl)]
   }
+
+  /// Debug description of this lookup name.
+  @_spi(Experimental) public var debugDescription: String {
+    let sourceLocationConverter = SourceLocationConverter(fileName: "", tree: syntax.root)
+    let location = sourceLocationConverter.location(for: position)
+    let strName = (identifier != nil ? identifier!.name : "NO-NAME") + " at: \(location.line):\(location.column)"
```

Comment:
How about using `??` and string interpolation

```suggestion
    let strName = "\(identifier?.name ?? "NO-NAME") at: \(location.line):\(location.column)"
```